scriptoocloud/pom.xml

```diff
@@ -1,324 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we need this file (actually nothing will work without it), please don't remove it.

Reply:
Readded.